packages/initiatives/src/update-initiative-site-id.ts

```diff
@@ -0,0 +1,62 @@
+/* Copyright (c) 2020 Environmental Systems Research Institute, Inc.
+ * Apache-2.0 */
+import { IItem } from "@esri/arcgis-rest-portal";
+import { IModel, IHubRequestOptions, getProp, isGuid } from "@esri/hub-common";
+import { updateItem, getItem } from "@esri/arcgis-rest-portal";
+
+/**
+ * Update the Site associated with an Initiative by setting the
+ * `item.properties.siteId` to a new value
+ *
+ * Used during createSite(...) and any time we need to
+ * connect a different Site to an Initiative
+ *
+ * @param initiativeItemId string | IItem | IModel
+ * @param siteId string
+ * @param hubRequestOptions IHubRequestOptions
+ */
+export function updateInitiativeSiteId(
+  maybeModel: string | IItem | IModel,
```

Comment:
Why the flexibility?

Reply:
in Hub.js we will be passing in a Guid, but from w/in Hub, where we want to "heal" this, we may have the Item or Model already - thus allowing us to skip the fetch